client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -200,30 +201,17 @@ class GSuiteAddUsers extends React.Component {
 
 		return (
 			<Fragment>
-				{ domainsWithForwards.length ? (
-					<Notice showDismiss={ false } status="is-warning">
-						{ translate(
-							'Please note that email forwards are not compatible with %(productFamily)s, and will be disabled once %(productFamily)s is added to this domain. The following domains have forwards:',
-							{
-								args: { productFamily: getGoogleMailServiceFamily() },
-								comment: '%(productFamily)s can be either "G Suite" or "Google Workspace"',
-							}
-						) }
-						<ul>
-							{ domainsWithForwards.map( ( domainName ) => {
-								return <li key={ domainName }>{ domainName }</li>;
-							} ) }
-						</ul>
-					</Notice>
-				) : (
-					''
+				{ userCanPurchaseGSuite && isNewAccount && (
+					<Fragment>
+						<EmailExistingForwardsNotice
+							domainsWithForwards={ domainsWithForwards }
+							selectedDomainName={ selectedDomainName }
+						/>
+
+						<QueryEmailForwards domainName={ selectedDomainName } />
```

Comment:
I could be in the minority here, but I would have expected this component to be located above the notice since the latter relies on the data retrieved by the former.